src/inverseRobot.js

```diff
@@ -7,7 +7,23 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  if (!robot) {
+    return null;
+  }
+
+  const formattedRobot = {};
+
+  for (const key in robot) {
+    const newKey = robot[key];
```

Comment:
1) [VARIABLES] - use variables for the main values so that you'll be able to reuse them and give them descriptive names.
For example, you can name them
```
    const INCOMING_ROBOT_VALUE = robot[key];
    const INCOMING_ROBOT_KEY = key;
```

Reply:
On line 16, there is already a declaration of the key variable. On line 17 is a declaration of the value variable=/ I don't think upper case for variables is needed here. https://www.freecodecamp.org/news/when-to-capitalize-your-javascript-constants-4fabc0a4a4c4/